app/scripts.babel/common.js

```diff
@@ -13,36 +16,60 @@ function ajax(options, callback) {
   return request.send(options.data);
 }
 
-function getSiteLocale() {
-  return /Задачи/.test(document.querySelector('body').innerText) ? 'ru' : 'en';
-}
-
 let messageBundle;
 
 function loadMessageBundle(callback) {
-  const locale = getSiteLocale();
-  ajax({
-    method: 'GET',
-    url: chrome.runtime.getURL(`_locales/${locale}/messages.json`),
-  }, response => {
-    messageBundle = JSON.parse(response);
+  getSiteMessages(messages => {
+    messageBundle = messages;
     callback();
   });
 }
 
 function getMessage(id) {
-  return messageBundle[id].message;
+  return messageBundle[id];
+}
+
+const FRIENDS_KEY = 'friends';
+const FRIENDS_UPDATED_KEY = 'friends_updated';
+const LOCALSTORAGE_EXPIRE_SECS = 5 * 60;
+
+function saveFriendsToLocalStorage(friends) {
+  localStorage[FRIENDS_UPDATED_KEY] = Date.now();
```

Comment:
Note that `localStorage` stores everything as strings. Use `setItem` and `getItem` to make it more obvious.